hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/TestCapacitySchedulerAmbiguousLeafs.java

```diff
@@ -67,13 +68,10 @@ private ApplicationId submitApplication(MockRM rm, String queue)
 
   @Test
   public void testAmbiguousSubmissionWithACL() throws Exception {
-    YarnConfiguration conf = new YarnConfiguration();
-    conf.set(YarnConfiguration.RM_SCHEDULER, CapacityScheduler.class.getName());
-    conf.setBoolean(YarnConfiguration.YARN_ACL_ENABLE, true);
+    CapacitySchedulerConfiguration schedulerConf = new CapacitySchedulerConfiguration();
 
-    MockRM rm = new MockRM(conf);
-    CapacityScheduler cs = (CapacityScheduler)rm.getResourceScheduler();
-    CapacitySchedulerConfiguration schedulerConf = cs.getConfiguration();
+    schedulerConf.set(YarnConfiguration.RM_SCHEDULER, CapacityScheduler.class.getName());
+    schedulerConf.setBoolean(YarnConfiguration.YARN_ACL_ENABLE, true);
```

Comment:
Nit: I know the new code is technically similar (because CSConf inherits YarnConf's properties), but RM_SCHEDULER and YARN_ACL_ENABLE are YarnConfiguration entries. For that reason I think the previous implementation - while more verbose - looks better. The reason for the change is probably the relocated setupCapacityScheduler call, but schedulerConf could be created with the `public CapacitySchedulerConfiguration(Configuration configuration)` constructor from YarnConfiguration.